DCode.Web/Scripts/App_JS/Controllers/Contributor.js

```diff
@@ -87,12 +92,16 @@
                 setTimeout(function () { $('#txtManagerEmailId' + index).focus() }, 1);
                 //$location.hash('div' + index);
             }
-            else if (task.TypeId >= 2)
-            {
+            else if (task.TypeId >= 2) {
                 $scope.reviewIndex = index;
-                $scope.applyFITask(task)
+                if (undefined == task.SelectedProfType || '' == task.SelectedProfType) {

```

Comment:
`undefined == task.SelectedProfType` should be re-written as `task.SelectedProfType==undefined` as we are to compare a value against an expected one, and not the other way around. Same for the other boolean evaluation.